examples/hello_world/client/nodejs/index.js

```diff
@@ -1,24 +1,54 @@
 const fs = require('fs');
-const protoLoader = require('@grpc/proto-loader');
+const protobufjs = require('protobufjs');
 const grpc = require('@grpc/grpc-js');
+const grpcProtoLoader = require('@grpc/proto-loader');
 
 const CERT_PATH = __dirname + '/../../../certs/local/ca.pem';
-const PROTO_PAH = __dirname + '/../../proto/hello_world.proto';
-
-const packageDefinition = protoLoader.loadSync(PROTO_PAH);
-const helloWorldProto = grpc.loadPackageDefinition(packageDefinition).oak
-  .examples.hello_world;
-const credentials = grpc.credentials.createSsl(fs.readFileSync(CERT_PATH));
-
-// TODO(#1066): Use a more restrictive Label.
-const client = new helloWorldProto.HelloWorld('localhost:8080', credentials);
-
-client.sayHello({ greeting: 'Node.js' }, (error, response) => {
-  if (error) {
-    console.error(error);
-    process.exit(1);
-  } else {
-    console.log(response.reply);
-    process.exit(0);
+const SERVICE_PROTO_PAH = __dirname + '/../../proto/hello_world.proto';
+const OAK_LABEL_PROTO_PAH = __dirname + '/../../../../oak/proto/label.proto';
```

Comment:
s/PAH/PATH/ x 2

Reply:
What do you mean by this?

Reply:
I meant that `SERVICE_PROTO_PAH` looks like a typo for `SERVICE_PROTO_PATH`, similarly for `OAK_LABEL_PROTO_PAH`, so replace PAH with PATH?

Reply:
FYI https://www.gnu.org/software/sed/manual/html_node/The-_0022s_0022-Command.html

s/old/new/ is a common pattern to indicate replacement

Reply:
ah yes, thanks! These obvious things are always invisible to me.

Reply:
[See also](https://github.com/project-oak/oak/pull/1017#discussion_r430308804)

Reply:
@daviddrysdale caught in my crimes